packages/robo/src/cli/utils/cloudflared.ts

```diff
@@ -73,22 +152,72 @@ export function isCloudflaredInstalled(to = DEFAULT_BIN_PATH): boolean {
 	return fs.existsSync(to)
 }
 
+export async function initializeCloudflareTunnel(): Promise<boolean> {
+	if (!process.env.CLOUDFLARE_API_KEY || !process.env.CLOUDFLARE_ACCOUNT_ID || !process.env.CLOUDFLARE_DOMAIN) {
+		cloudflareLogger.warn('Missing required Cloudflare environment variables.');
+		return false;
+	}
+
+	try {
+		if (process.env.CLOUDFLARE_TUNNEL_ID) {
+			const roboTunnel = await cloudflareRequest(`/accounts/${process.env.CLOUDFLARE_ACCOUNT_ID}/cfd_tunnel/${process.env.CLOUDFLARE_TUNNEL_ID}/token`);
```

Comment:
Do Cloudflare's tunnel tokens expire at some point? If so, do we have a way to auto refresh them?